x-pack/plugins/task_manager/server/task_claimers/strategy_mget.ts

```diff
@@ -105,9 +106,9 @@ async function claimAvailableTasksApm(opts: TaskClaimerOpts): Promise<ClaimOwner
 async function claimAvailableTasks(opts: TaskClaimerOpts): Promise<ClaimOwnershipResult> {
   const { getCapacity, claimOwnershipUntil, batches, events$, taskStore, taskPartitioner } = opts;
   const { definitions, unusedTypes, excludedTaskTypes, taskMaxAttempts } = opts;
-  const { logger } = opts;
-  const loggerTag = claimAvailableTasksMget.name;
-  const logMeta = { tags: [loggerTag] };
+  const logger = createWrappedLogger({ logger: opts.logger, tags: [claimAvailableTasksMget.name] });
+  // const loggerTag = claimAvailableTasksMget.name;
+  // const logMeta = { tags: [loggerTag] };
```

Comment:
remove unneeded comments

Reply:
Whoops, fixed in https://github.com/elastic/kibana/pull/192574/commits/795b4d959d4e530e3132a5c4a9876360865df3f0